about.html

```diff
@@ -24,7 +24,7 @@ <h2>
       <div class="row justify-content-center text-center">
         <div class="col-sm col-lg-8">
           <h2>
-            Our core values
+            Core values

```

Comment:
Why remove our?

Reply:
Because it sounds like something from a job description or would normally follow "meet the team, who we are" type page of profile pics and job titles. 

Vowpal Wabbit values?